docs/cookbooks/implementators/secure_content_push.rst

```diff
@@ -0,0 +1,26 @@
+Secure content push to Publisher
+================================
+
+Content can be pushed to Publisher via HTTP requests from any system. But it's important store and publish one requests
+from approved by us sources.
```

Comment:
`But it's important store and publish one requests from approved by us sources.` - I don't get it